### PR TITLE
Activity member tab integration

### LINKED ISSUE #2472: Fetch data for Activity of Member tab on Member Profile Details
Pre-cursor: https://github.com/Joystream/atlas/issues/1804

### PR DESCRIPTION
**Pull request by drillprop**
Fix #2472 
Sorting and infinite scrolling aren't in the scope of this PR

### CODE REVIEW

**Comment by eldiegod** [comment]
lets do a rebase on this one

File: packages/atlas/src/views/viewer/MemberView/MemberActivity.hooks.ts [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+      from: BasicMembershipFieldsFragment | null
+    } & NftActivitiesRecord)
+  | ({
+      type: 'Price change'
+      from: BasicMembershipFieldsFragment | null
+      price: number
+    } & NftActivitiesRecord)
+
+const parseActivities = (
+  event: ReturnType<typeof useRawActivities>['activities'][0],
```
Comment (eldiegod) [inline packages/atlas/src/views/viewer/MemberView/MemberActivity.hooks.ts]: We don't need to grab the first item of the array we can just do: 
```suggestion
  event: ReturnType<typeof useRawActivities>['activities'][number],
```

**Review by eldiegod — APPROVED** [review]
looks pretty neat

**Review by kdembler — CHANGES_REQUESTED** [review]
We're missing price here:
Screenshot 2022-04-21 at 20 11 40

Also, what do we do with sorting? The select is still there but has no effect. Making it work is probably super easy since we do sorting client side anyway

**Review by kdembler — APPROVED** [review]

**Pull request merged**